src/testers.jl

```diff
@@ -132,40 +132,26 @@ end
 
 All keyword arguments except for `fdm` are passed to `isapprox`.
 """
-function rrule_test(f, ȳ, (x, x̄)::Tuple{Any, Any}; rtol=1e-9, atol=1e-9, fdm=_fdm, kwargs...)
+function rrule_test(f, ȳ, xx̄s::Tuple{Any, Any}...; rtol=1e-9, atol=1e-9, fdm=_fdm, kwargs...)
     _ensure_not_running_on_functor(f, "rrule_test")
 
     # Check correctness of evaluation.
-    fx, pullback = rrule(f, x)
-    @test collect(fx) ≈ collect(f(x))  # use collect so can do vector equality
-    (∂self, x̄_ad) = if fx isa Tuple
+    xs, x̄s = collect(zip(xx̄s...))
+    y_ad, pullback = rrule(f, xs...)
+    y = f(xs...)
+    # use collect so can do vector equality
+    @test isapprox(collect(y_ad), collect(y); rtol=rtol, atol=atol)
+    @assert !(isa(ȳ, Thunk))
+    ∂s = if y_ad isa Tuple
         # If the function returned multiple values,
         # then it must have multiple seeds for propagating backwards
         pullback(ȳ...)
     else
         pullback(ȳ)
     end
```

Comment:
Personally I'm a big fan of ternary operators, but I understand if it's not everyones cup of tea. I can go either way, if you don't like it just close the comment :) 

```suggestion
    # If the function returned multiple values,
    # then it must have multiple seeds for propagating backwards
    ∂s = (y_ad isa Tuple) ? pullback(ȳ...) : pullback(ȳ)
```

Reply:
I also like ternary operators; I was just trying to make the smallest possible number of changes.